src/test/java/core/basesyntax/service/FruitTransactionImplTest.java

```diff
@@ -0,0 +1,55 @@
+package core.basesyntax.service;
+
+import core.basesyntax.dao.FruitDao;
+import core.basesyntax.dao.FruitDaoImpl;
+import core.basesyntax.db.Storage;
+import core.basesyntax.model.Fruit;
+import core.basesyntax.strategy.MapStrategy;
+import java.util.List;
+import org.junit.Assert;
+import org.junit.Test;
+
+public class FruitTransactionImplTest {
+    private static final MapStrategy mapStrategy = new MapStrategy();
+    private static final FruitDao fruitDao = new FruitDaoImpl();
+    private FruitTransaction fruitTransaction = new FruitTransactionImpl();
```

Comment:
initialize them inside of a method with @BeforeClass

Reply:
:)